src/akamai/js/akamaiHD_flash.js

```diff
@@ -0,0 +1,1334 @@
+    attributes.id = playerId;
+    attributes.class = 'video';
+    attributes.preload = 'none';
+    attributes.style = 'position:absolute;';
+    // Combine the css object into a string for swfobject.
+    if (cssFromContainer.length) {
+      for (i in cssFromContainer) {
+        attributes.style += i + ":" + cssFromContainer[i] + "; ";
+      }
+    }
+    attributes.name = playerId;
+    attributes.align = "middle";
+    swfobject.embedSWF(
+      pluginPath, playerId,
+      "100%", "100%",
+      flashMinimumVersion, flexPath,
+      flashvars, params, attributes, this.subscribeAllEvents);
+
+    JFlashBridge.bind(playerId, this);
+
+    var flashVideoObject = JFlashBridge.getSWF(playerId);
+    var readyToPlay = false; // should be set to true on canplay event
+    var actionscriptCommandQueue = [];
+
+    /************************************************************************************/
+    // Required. Methods that Video Controller, Destroy, or Factory call
+    /************************************************************************************/
+
+    /**
+     * Subscribes to all events raised by the video element.
+     * This is called by the Factory during creation.
+     * @public
+     * @method OoyalaFlashVideoWrapper#subscribeAllEvents
+     */
+    this.subscribeAllEvents = function() {
+      listeners = { "play": _.bind(raisePlayEvent, this),
+                    "playing": _.bind(raisePlayingEvent, this),
+                    "ended": _.bind(raiseEndedEvent, this),
+                    "error": _.bind(raiseErrorEvent, this),
+                    "seeking": _.bind(raiseSeekingEvent, this),
+                    "seeked": _.bind(raiseSeekedEvent, this),
+                    "pause": _.bind(raisePauseEvent, this),
+                    "ratechange": _.bind(raiseRatechangeEvent, this),
+                    "stalled": _.bind(raiseStalledEvent, this),
+                    "volumechange": _.bind(raiseVolumeEvent, this),
+                    "volumechangeNew": _.bind(raiseVolumeEvent, this),
+                    "waiting": _.bind(raiseWaitingEvent, this),
+                    "timeupdate": _.bind(raiseTimeUpdate, this),
+                    "durationchange": _.bind(raiseDurationChange, this),
+                    "loadstart": _.bind(onLoadStart, this),
+                    "loadedmetadata": _.bind(onLoadedMetadata, this),
+                    "progress": _.bind(raiseProgress, this),
+                    "canplaythrough": _.bind(raiseCanPlayThrough, this),
+                    "webkitbeginfullscreen": _.bind(raiseFullScreenBegin, this),
+                    "webkitendfullscreen": _.bind(raiseFullScreenEnd, this),
+                    "totalavailablebitrates": _.bind(raiseBitratesAvailable, this),
+                    "changedbitrate": _.bind(raiseBitrateChanged, this),
+                    "sizeChanged": _.bind(raiseSizeChanged, this),
+                    "fullscreenChanged": _.bind(raiseSizeChanged, this)
```

Comment:
Doe the functions need to exist at all?  What are they for?
Since this is a bare template let's make sure it only includes required functions to ensure that extraneous functions don't end up stuck in the code later